src/types/precomp.h

```diff
@@ -29,6 +29,7 @@ Module Name:
 
 // Windows Header Files:
 #include <windows.h>
+#include <dwmapi.h>

```

Comment:
We probably don't need this one for the moment now that ThemeUtils is stripped, right?